scripts/jenkins/Makefile.jenkins

```diff
@@ -29,15 +29,15 @@ test-package-py:
 	zip -q -r test-package-py.zip h2o-py/tests/ h2o-py/demos/ \
 	h2o-docs/src/booklets/v2_2015/source h2o-py/dist/*.whl \
 	h2o-genmodel/build/libs/h2o-genmodel.jar h2o-assemblies/genmodel/build/libs/genmodel.jar scripts/run.py \
-	$(JENKINS_MAKEFILE_PATH) tests/pyunitSmokeTestList h2o-py/scripts/h2o-py-test-setup.py
+	$(JENKINS_MAKEFILE_PATH) tests/pyunitSmokeTestList tests/pyunitAutoMLList h2o-py/scripts/h2o-py-test-setup.py
 
 test-package-r:
 	cd h2o-py/tests/ && rm -f -r testdir_hdfs
 	zip -q -r test-package-r.zip h2o-r/h2o-package/R/ h2o-r/h2o-package/**/*.Rd h2o-r/h2o-package/DESCRIPTION \
 		h2o-r/tests/ h2o-r/demos/ h2o-r/h2o-package/demo \
 		h2o-docs/src/booklets/v2_2015/source h2o-r/R/src/contrib/h2o*.tar.gz \
 		h2o-assemblies/genmodel/build/libs/genmodel.jar scripts/run.py \
-		$(JENKINS_MAKEFILE_PATH) tests/runitSmokeTestList \
+		$(JENKINS_MAKEFILE_PATH) tests/runitSmokeTestList tests/runitAutoMLList \
```

Comment:
This is probably fine for now but in the future, it would be good to have this dynamic.

Add eg. `--excludegroups` to run.py